src/elements/common/modal.scss

```diff
@@ -1,6 +1,4 @@
-@import '../../styles/variables';
 @import './variables';
-@import '../../styles/mixins/buttons';
```

Comment:
the `mixins/buttons` import is included in `styles/variables`
the `styles/variables` import is included in `elements/common/variables`